### DISCUSSION TITLE
Full list of all available Markdown options

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#23827] [New to GitHub]
<p>I was wondering, if there is a page, repository or similar with a complete (and I mean complete) list of all Markdown features GitHub offers, being it direct markdown features (formatting) or GitHub-only features (links to commits/SHA refernces)</p>
<p>The only pages I found are <a href="https://guides.github.com/features/mastering-markdown/" rel="nofollow noopener">https://guides.github.com/features/mastering-markdown/</a> and <a href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet" rel="nofollow noopener">https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet</a></p>
<p>The first one only lists the basic features GitHub provides and the second one does this too with some small alterations and additions.</p>
<p>There are quite a few features that are missing from the page and are never mentioned anywhere.</p>
<p>For example did I randomly discover that you can put a hex color code in inline code to then display a small icon next to it.</p>
<p>Example:  <img src="https://cdn.discordapp.com/attachments/536585380389191720/711175934027497512/unknown.png" alt=""></p>
<p>Additionally does the markdown page display outdated(?) formatting? Like f.e. does an ordered list with an intend unordered list (no longer) work with just two spaces and requires three/four spaces.</p>
<p>Tl;dr The cheatsheet misses a lot of useful information that people might wanna use when opening issues or creating wiki pages, which is sad.</p>
<p>So I hope there is an actual page containing all (known) markdown and GH-markdown features to use.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/andre601">@andre601</a>, Thanks for being here! And thanks for your feedback, I’ve taken your suggestion and passed it along to the appropriate teams.  Meanwhile, here is a <a href="https://github.github.com/gfm/" rel="nofollow noopener">link</a> to an unofficial guide a user put together, it might help.</p>

**Comment by Andre601** [comment]
<p>Thanks.</p>
<p>I went ahead and also made <a href="https://Andre601.github.io/GitHub-Markdown" rel="nofollow noopener"><strong>my very own version</strong></a> of useful GitHub (markdown) features, which also includes stuff that isn’t mentioned on the page you linked.</p>
